### ISSUE TITLE
CHR field misincorporation.txt

### ISSUE THREAD

**Issue by MeriamGuellil** [ginolhac/mapDamage#41]
Hi,
I noticed that the CHR field has been removed from the misincorporation.txt file. I was using it to plot damage patterns for the first few bases for each BAM header. Any chance we could either get this column back or get the information from the current version of mapdamage?

**Comment by ginolhac** [comment]
true, this is no longer there. @MikkelSchubert  could you do something it?

**Comment by MikkelSchubert** [comment]
This functionality is still available in the last tagged release (v2.2.1), but was removed in the mater branch due to a lack of perceived benefit and due to the problems it causes with poorly assembled genomes (extreme memory usage and gigantic table files). I'll look at re-adding it as an optional (off by default) feature when I get back to working on the mapDamage.
